app/grants/utils.py

```diff
@@ -28,6 +29,10 @@
 
 logger = logging.getLogger(__name__)
 
+block_codes = ['▖', '▗', '▘', '▙', '▚', '▛', '▜', '▝', '▞', '▟']
+emoji_codes = ['🎉', '🎈', '🎁', '🎊', '🙌', '🥂', '🎆', '🔥', '⚡', '👍']
```

Comment:
^ what are these for?

Reply:
This is used to create one id for every user, instead of using numbers it uses emojis. Why? Because if we use a generic text to verify grants, another user can create a grant with the same twitter account and click verify. So we'll have two accounts verified, adding the code limit this type of vulnerability.